src/dmd/expression.d

```diff
@@ -4989,6 +4989,31 @@ extern (C++) final class DotTemplateInstanceExp : UnaExp
         return ti.updateTempDecl(sc, s);
     }
 
+    override bool checkType()
```

Comment:
Please update the headers and frontend.h

Reply:
I always forget headers.
Done.